app/src/main/java/com/sun_asterisk/moviedb_49/data/model/Genres.kt

```diff
@@ -1,6 +1,17 @@
 package com.sun_asterisk.moviedb_49.data.model
 
 data class Genres(
-    var id: Int,
-    var name: String
-)
+    var name: String? = null
```

Comment:
Sao class có 1 thuộc tính lại dùng mẫu Builder, trong Kotlin có thể dùng những giá trị default của thuộc tính để giải quyết bài toán thay cho Builder Pattern